src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts

```diff
@@ -57,29 +57,27 @@ class AccessibleBufferContribution extends DisposableStore implements ITerminalC
 }
 registerTerminalContribution(AccessibleBufferContribution.ID, AccessibleBufferContribution);
 
-registerTerminalAction({
-	id: TerminalCommandId.ShowTerminalAccessibilityHelp,
-	title: { value: localize('workbench.action.terminal.showAccessibilityHelp', "Show Terminal Accessibility Help"), original: 'Show Terminal Accessibility Help' },
-	keybinding: {
-		primary: KeyMod.Alt | KeyCode.F1,
-		weight: KeybindingWeight.WorkbenchContrib,
-		linux: {
-			primary: KeyMod.Alt | KeyMod.Shift | KeyCode.F1,
-			secondary: [KeyMod.Alt | KeyCode.F1]
-		},
-		when: TerminalContextKeys.focus
-	},
-	run: async (c, accessor: ServicesAccessor) => {
-		const instantiationService = accessor.get(IInstantiationService);
-		const instance = await c.service.getActiveOrCreateInstance();
-		await c.service.revealActiveTerminal();
-		const terminal = instance?.xterm;
-		if (!terminal) {
-			return;
+export class TerminalAccessibilityHelpContribution extends Disposable {
+	static ID: 'terminalAccessibilityHelpContribution';
+	constructor() {
+		super();
+
+		if (AccessibilityHelpAction) {
+			this._register(AccessibilityHelpAction.addImplementation(105, 'terminal', async accessor => {
+				const instantiationService = accessor.get(IInstantiationService);
+				const terminalService = accessor.get(ITerminalService);
+				const instance = await terminalService.getActiveOrCreateInstance();
+				await terminalService.revealActiveTerminal();
+				const terminal = instance?.xterm;
+				if (!terminal) {
+					return;
```

Comment:
This the best way to handle no xterm?

Reply:
If it's not yet attached, it's not focused so will fail to get run anyway bc of `when: TerminalContextKeys.focus`